linera-views/tests/store_tests.rs

```diff
@@ -248,9 +271,12 @@ async fn test_indexed_db_big_write_read() {
 #[cfg(with_dynamodb)]
 #[tokio::test]
 async fn test_dynamo_db_big_write_read() {
+    use linera_views::store::AdminKeyValueStore as _;
+
     let store = linera_views::dynamo_db::DynamoDbStore::new_test_store()
         .await
         .unwrap();
+    let store = store.clone_with_root_key(&[]).unwrap();
```

Comment:
I do not think that is needed as the `run_big_write_read` is very much not cached.

Reply:
This seems to be required:
https://github.com/linera-io/linera-protocol/actions/runs/13734903398/job/38417381624

Reply:
It's more about unsafe journaling actually